proof/crefine/RISCV64/TcbAcc_C.thy

```diff
@@ -120,13 +120,12 @@ lemma register_from_H_less:
   by (cases hr, simp_all add: "StrictC'_register_defs")
 
 lemma register_from_H_sless:
-  "register_from_H hr <s 35"
+  "UCAST(8 \<rightarrow> 32 signed) (register_from_H hr) <s 0x23"
   by (cases hr, simp_all add: "StrictC'_register_defs" word_sless_def word_sle_def)
 
-lemma register_from_H_0_sle[simp]:
-  "0 <=s register_from_H hr"
-  using word_0_sle_from_less[OF order_less_le_trans] register_from_H_less
-  by fastforce
+lemma register_from_H_0_sle'[simp]:
+  "0 <=s UCAST(8 \<rightarrow> 32 signed) (register_from_H hr)"
```

Comment:
Same as above, I'd give `0` the type, and only write lower-case `ucast` on the right.